yarn.lock

```diff
@@ -8711,7 +8716,7 @@ release-zalgo@^1.0.0:
   dependencies:
     es6-error "^4.0.1"
 
-request@2.88.2, request@^2.88.2:
+request@2.88.2:
```

Comment:
This is an indirect dev dependency, so it will not block any new npmjs packages as its code is not released. It is used by `web-ext` which is used to build firefox extension. Mozilla team has plans to remove this dependency too:

https://github.com/mozilla/web-ext/issues/2130#issuecomment-1168408414